integration_tests/tests/apply/submitted_applications/view_application_overview_as_referrer.cy.ts

```diff
@@ -41,13 +46,19 @@ context('View submitted application overview', () => {
               label: 'Awaiting decision',
               description: 'All information has been received and the application is awaiting assessment.',
             }),
-            statusUpdateFactory.build({
-              updatedAt: DateFormats.dateObjToIsoDateTime(faker.date.past()),
-              updatedBy: externalUserFactory.build({ name: 'Anne Assessor' }),
-              name: 'moreInfoRequested',
+          ],
+          timelineEvents: [
+            timelineEventsFactory.build({
+              occurredAt: DateFormats.dateObjToIsoDateTime(faker.date.future()),
+              createdByName: 'Anne Other Assessor',
+              label: 'Awaiting decision',
+              body: 'All information has been received and the application is awaiting assessment.',
```

Comment:
Minor but a status update will only have a `body` of status details at the moment, so could update test data to reflect that.

Reply:
aware that's not really codified anywhere 😢

Reply:
> Minor but a status update will only have a body of status details at the moment, so could update test data to reflect that.

@Harriethw I don't quite follow, could you give me an example?